index.bs

```diff
@@ -9039,7 +9039,7 @@ specifying headers that will extend, or overwrite, existing request headers.
       )
 
       network.SetExtraHeadersParameters = {
-        headers: [+network.Header]
+        headers: [network.Header]
```

Comment:
```suggestion
        headers: [*network.Header]
```

I quickly scanned the repo and it's either [+something] or [*something], so let's use `*` here.

Reply:
done, thanks!